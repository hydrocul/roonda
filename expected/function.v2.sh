#!/bin/sh

#################################################
# roonda_b24fff3df8ea4a7cf535c39191317e9c07720733.py:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/roonda_b24fff3df8ea4a7cf535c39191317e9c07720733.py
import sys

x = 10
def f (a, b):
    x = a + b
    print(str(x))

def g ():
    print(str(x))

f(3, 4)
g()
print(str(x))
END_OF_ROONDA_SOURCE_FILE
#################################################

echo python
python2 $ROONDA_TMP_PATH/roonda_b24fff3df8ea4a7cf535c39191317e9c07720733.py
