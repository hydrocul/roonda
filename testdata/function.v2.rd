sh v2

(println "python")
(python2
 (assign x 10)
 (function f (a b)
           (assign x (+ a b))
           (println x))
 (function g ()
           (println x))
 (f 3 4)
 (g)
 (println x))

