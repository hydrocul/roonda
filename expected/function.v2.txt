python
7
10
10
